src/libraries/CurveLib.sol

```diff
@@ -28,7 +27,8 @@ interface ICurvePoolLike is IERC20 {
     function remove_liquidity(
         uint256 burnAmount,
         uint256[] memory minAmounts,
-        address receiver
+        address receiver, 
+        bool claimAdminFees
```

Comment:
Curve defaults to `true` for this, but it seems from the tests that this can affect the tokens returned in low liquidity pools.  I think it is better to use `false` here

Reply:
Can you provide an example of when and why it makes a difference? We can take this offline.